tests/cli/bad-component-new.wat.stderr

```diff
@@ -4,4 +4,4 @@ Caused by:
     0: failed to decode world from module
     1: module was not valid
     2: failed to resolve import `a::a`
-    3: module requires an import interface named `a`
+    3: module requires an import interface named `a`
```

Comment:
Could you try rebasing on `main` and re-BLESS-ing tests to remove some of these changes?